oat-sa/extension-tao-itemqti
# Feature/ADF-1744/disable instructions

### Related to: https://oat-sa.atlassian.net/browse/ADF-1744

### Summary

Add support for instructions hiding based on a custom CSS class set at the item level.

### Details

When an author sets the class `remove-instructions` or `__custom__remove-instructions` in the item element, the instructions or constraints using the feedback blocks should be hidden.

An author can activate such an option from the item properties. The `Remove Instruction` option is off by default.
![image](https://github.com/user-attachments/assets/3870ddfc-7d72-4d78-a1bf-8d808f6e8cf8)


### How to test

Note: To see the change in the NewUI previewer, please look at the companion PR.

- check out the branch: `git checkout -t origin/ feature/ADF-1744/disable-instructions`
- open an item and add some constraints: for example, a choice interaction with min and max selection
- open the item in the previewer and look at the instructions and constraints 
![image](https://github.com/user-attachments/assets/a105a08b-e1e0-4f7a-8b62-755a6cf373dc)
![image](https://github.com/user-attachments/assets/1462e3fc-a1a4-40b3-af08-16f0e71414d3)
- check the option `Remove Instruction` at the item properties panel
- open the item in the previewer and see that the instructions and constraints are not presented

### Notes
Some files were auto-formatted and this may add noise to the review. Consider activating the `Hide whitespaces` feature in GH, to limit the side effects.
![image](https://github.com/user-attachments/assets/f47756bb-d4fa-48a4-b990-6085a3ec3b3d)
